Toolbox/scripts/sshd_install.sh

```diff
@@ -0,0 +1,99 @@
+#!/bin/sh
+
+SSD_DIR=/net/mmx/mnt/app/eso/hmi/engdefs/scripts/ssh
+
+export LD_LIBRARY_PATH=/net/mmx/mnt/app/root/lib-target:/net/mmx/eso/lib:/net/mmx/mnt/app/usr/lib:/net/mmx/mnt/app/armle/lib:/net/mmx/mnt/app/armle/lib/dll:/net/mmx/mnt/app/armle/usr/lib
+export PATH=.:${SSD_DIR}/usr/bin:${SSD_DIR}/usr/sbin:/proc/boot:/bin:/usr/bin:/usr/sbin:/sbin:/mnt/app/media/gracenote/bin:/mnt/app/armle/bin:/mnt/app/armle/sbin:/mnt/app/armle/usr/bin:/mnt/app/armle/usr/sbin
+
+#Is there any SD-card inserted?
+if [ -d /net/mmx/fs/sda0 ]; then
+    echo SDA0 found
+    export VOLUME=/net/mmx/fs/sda0
+elif [ -d /net/mmx/fs/sdb0 ]; then
+    echo SDB0 found
+    export VOLUME=/net/mmx/fs/sdb0
+else 
+    echo No SD-cards found.
+    exit 0
+fi
```

Comment:
Why do you use this insted of the commeted out function from line 20 to 25?

Reply:
Ah, I have it here in long form so I could run the script on console direct from SD card - toolbox isn't installed on my bench unit yet :-D